package.json

```diff
@@ -65,6 +65,7 @@
     "enzyme-to-json": "3.6.1",
     "immer": "7.0.14",
     "mock-socket": "^9.0.3",
+    "nanoid": "3.1.16",
```

Comment:
Is this required? Should react-components not be pulling it in as its own dependency?

Reply:
It should but it currently does not: https://github.com/canonical-web-and-design/react-components/issues/327

When they fix that, we can drop this dep.